cargodocs/centralized.md

```diff
@@ -0,0 +1,110 @@
+Telescope construction with prehashing and bounded DFS scheme given in _Section 3.2.2_.
+
+
+In some cases, $n_p$ may be small, which can affect the completeness and efficiency of the proof construction. The protocol adapts to such cases by introducing optimizations and tighter bounds.
```

Comment:
@tolikzinovyev :  If we are to give a name of `n_p` in English, what would it be? I still think of it as the cardinality of `S_p` and don't understand why it is a separate parameter.

Reply:
In the paper we simply defined n_p = |S_p|. It's possible to consider a more general case: |S_p| >= n_p. Then we can maybe call n_p the required prover set size.